src/knexfile.ts

```diff
@@ -0,0 +1,43 @@
+import type { Knex } from "knex";
+
+// Update with your config settings.
+
+const config: { [key: string]: Knex.Config } = {
+  development: {
+    client: "postgresql",
+    connection: {
+      database: process.env.POSTGRES_DB,
+      user: process.env.POSTGRES_USER,
+      password: process.env.POSTGRES_PASSWORD,
+      host: process.env.POSTGRES_HOST
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: "knex_migrations",
+      directory: "./migrations"
+    }
+  },
+
+  production: {
```

Comment:
I don't think we will ever differentiate this, because we should control it with env var